Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -59,19 +59,24 @@ extern "C" void RunApplicationMain(Platform::String^ principalClassName,
                                    float windowHeight,
                                    ActivationType activationType,
                                    Platform::Object^ activationArg) {
+    // Return from activation immediatly; push all subsequent app launch work to 
```

Comment:
Agreed; it's an additional 'fix' for the app launch timeouts.  Raj had intended to do this with a previous change, but it didn't push all work off of the initial activation.  I agree that we need to test adequately.